ops/framework.py

```diff
@@ -871,6 +874,11 @@ def _emit(self, event: EventBase):
         saved = False
         event_path = event.handle.path
         event_kind = event.handle.kind
+        ops_event = event.__class__.__module__.startswith('ops.')
+        opentelemetry.trace.get_current_span().add_event(
+            f'{"ops." if ops_event else ""}{event.__class__.__name__}',
```

Comment:
The inline if/else inside a f-string variable confused me a lot. Plus, this looks like it's simply concatenating type strings, so what about `('ops.' if ops_event else '') + event.__class__.__name__`. 

Also, why do we only add the module prefix if it's in ops?